fe1-web/network/OperationError.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * This class allows us to distinguish operation errors
+ * (server sending back and error) from other JS errors.
+ */
+export class OperationError extends Error {
+  public readonly errorCode: number;
+
+  constructor(m: string, errorCode: number) {
+    super(m);
+    this.errorCode = errorCode;
+
+    // set prototype explicitly, needs to happen immediately after super() call
```

Comment:
the code violates the comment... you can put line 10 at the end of the constructor !

Reply:
woops!